cluster/selfplay/tpu-player-deployment-nr.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: tpu-player-deployment-nr
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: tpu-player
+  template:
+    metadata:
+      annotations:
+        tf-version.cloud-tpus.google.com: "1.11"
+      labels:
+        app: tpu-player
+    spec:
+      containers:
+      - name: minigo-tpu-player-container
+        image: gcr.io/tensor-go/minigo-tpu-player:latest_nr
+        imagePullPolicy: Always
+        resources:
+          limits:
+            cloud-tpus.google.com/v2: 8
```

Comment:
can/should this be parameterized?

Reply:
i don't think so.  I think these are scripts that people run cautiously, because they cost $$, so they should probably read and understand them first.